carbonserver/trie.go

```diff
@@ -467,6 +467,13 @@ func (tn *trieNode) setChild(i int, n *trieNode) {
 	)
 }
 
+func (tn *trieNode) incrementFindMetric() {
+	if tn.file() {
+		meta := tn.meta.(*fileMeta)
```

Comment:
just to be on safe side, let's also check if meta isn't nil:

```
meta, ok := tn.meta.(*fileMeta)
if !ok { return }
```

Reply:
sorry, scratch this, I think it's not really needed as we always create file trieNode with fileMeta.